src/components/ProjectDetailPage/ProjectDetailPage.tsx

```diff
@@ -43,15 +43,27 @@ export const ProjectDetailPage = () => {
   return (
     <DataContainer {...dataContainer} title="Project Details">
       <PageLayout title={dataContainer.data?.name} description={dataContainer.data?.description}>
-        <SectionHeader actions={<ActionButton link="edit" actionType="edit" />} />
+        <SectionHeader
+          actions={
+            <ActionButton link="edit" iconType="edit">
+              Edit
+            </ActionButton>
+          }
+        />
         <div className="m-b-25">
           <Card>
             <CardBody>
               <AttributesItems attributes={attributes} />
             </CardBody>
           </Card>
         </div>
-        <SectionHeader actions={<ActionButton action={() => console.log('Not implemented yet!')} actionType="create" />}>
+        <SectionHeader
+          actions={
+            <ActionButton action={() => console.log('Not implemented yet!')} iconType="create">
```

Comment:
Actually Project Create page is already implemented: `/projects/create`

Reply:
I think we need to use a link to a BuildConfig create page tho.

Reply:
you are right! Deleting this comment, as it's not valid